README.md

```diff
@@ -73,6 +73,15 @@ archivematica_src_mcpclient_clamav_tcp_ip: "1.2.3.4"
 archivematica_src_mcpclient_clamav_tcp_port: "3310"
 ```
 
+Configure users and register pipeline
+-------------------------------------
+
+A Storage Service user can be added with the role variable `archivematica_src_configure_ss` (default `no`).
```

Comment:
storage service user *must* be added (at least on 1.8), there is no default user created